optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -453,9 +453,9 @@ public int hashCode() {
                             it.remove();
                         }
                     }
-                    newResult.entityCount = ConfigUtils.mergeProperty(oldResult.entityCount, newResult.entityCount);
-                    newResult.variableCount = ConfigUtils.mergeProperty(oldResult.variableCount, newResult.variableCount);
-                    newResult.problemScale = ConfigUtils.mergeProperty(oldResult.problemScale, newResult.problemScale);
+                    newResult.entityCount = ConfigUtils.meldProperty(oldResult.entityCount, newResult.entityCount);
```

Comment:
Makes sense. Would you mind adding a comment to indicate that meld is used to deal with failed singles? I 'll already merge - just add it to another PR.